### PR TITLE
general flss loans query

### PR DESCRIPTION
**Pull request by go-bears**
Jira Ticket:
https://kiva.atlassian.net/browse/PSD-265

What does this do:
creates graphql loan query to flss service with filtering ability. the query should return all information necessary to populate a loan card. 

how to test:
The full expanded version of the query with a country default of ["US"] should return loans from the flss endpoint:
`https://marketplace-api.k1.kiva.org/graphql`

```graphql
fragment loanCardFields on LoanBasic {
	id
	status
	name
	borrowerCount
	geocode {
		# city added for DetailedLoanCard Query Cache
		city
		# state is used for US loans on LoanChannelCategoryPage
		state
		country {
			isoCode
			name
			region
		}
	}
	use
	activity {
		id
		name
	}
	sector {
		id
		name
	}
	whySpecial
	lenderRepaymentTerm
	loanAmount
	minNoteSize
	loanFundraisingInfo {
		fundedAmount
		reservedAmount
		isExpiringSoon
	}
	plannedExpirationDate
	matchingText
	userProperties {
		favorited
		lentTo
	}
	# lenders added for DetailedLoanCard Query Cache
	lenders(limit: 0) {
		totalCount
	}
	... on LoanPartner {
		partnerName
	}
	...on LoanDirect {
		trusteeName
	}
}

query flssLoanData(
  $filterObject: [FundraisingLoanSearchFilterInput!] = { countryIsoCode: {any: ["US"]}}
  $imgDefaultSize: String = "w480h360"
  $imgRetinaSize: String = "w960h720"
) {
  fundraisingLoans(filters: $filterObject) {
    totalCount
    values {
      id
      ...loanCardFields
      image {
        id
        default: url(customSize: $imgDefaultSize)
        retina: url(customSize: $imgRetinaSize)
        hash
      }
    }
  }
}
```

### CODE REVIEW

File: src/graphql/query/flssLoansQuery.graphql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#import '../fragments/loanCardFields.graphql'
+
+query flssLoanData(
+  $filterObject: [FundraisingLoanSearchFilterInput!]
+  $imgDefaultSize: String = "w480h360"
+  $imgRetinaSize: String = "w960h720"
+) {
```
Comment (mcstover) [inline src/graphql/query/flssLoansQuery.graphql:7]: Looks good so far. Please add in variable handling for the `limit` and `pageNumber` so we can query for a specific number of loans to show and handle paging too.

Comment (go-bears) [reply src/graphql/query/flssLoansQuery.graphql:7]: cool. thanks for 👀

Comment (go-bears) [reply src/graphql/query/flssLoansQuery.graphql:7]: so add
$limit: int = (is there a default value you like?)
$pageNumber: int (no particular default)?

Comment (mcstover) [reply src/graphql/query/flssLoansQuery.graphql:7]: We generally don't set defaults for these as the api should be defaulting to 20 for limit and 0 for pageNumber. Offset is how we usually term what is "pageNumber" in this case.

If there are no defaults in flss we can use 20 + 0

File: src/graphql/query/flssLoansQuery.graphql [hunk 2]
```diff
@@ -1,11 +1,15 @@
 #import '../fragments/loanCardFields.graphql'
 
 query flssLoanData(
-  $filterObject: [FundraisingLoanSearchFilterInput!]
+  $filterObject: [FundraisingLoanSearchFilterInput!] = { countryIsoCode: {any: ["US"]}}
   $imgDefaultSize: String = "w480h360"
   $imgRetinaSize: String = "w960h720"
+  $limit: Int = 50
```
Comment (mcstover) [inline src/graphql/query/flssLoansQuery.graphql:7]: This is too high... Two reasons, 1. we couldn't show that many loans on a single screen ( currenly we might show 9 on a big screen), 2. we would never want to try and ingest that many loans into a client at one time as all the extra data in loanCardFields has to resolve from graphql too which would create a massive data payload.

Comment (go-bears) [reply src/graphql/query/flssLoansQuery.graphql]: thanks for the insight, Matt! You got to the comment box before my next push went through 😄 
you're 💨  fast!

Comment (go-bears) [reply src/graphql/query/flssLoansQuery.graphql]: appreciate the eyes and the explanation!

File: src/graphql/query/flssLoansQuery.graphql (hunk 1 above)
Comment (mcstover) [reply src/graphql/query/flssLoansQuery.graphql:9]: Thank you!

**Review by mcstover — APPROVED** [review]

File: src/graphql/query/flssLoansQuery.graphql (hunk 1 above)
Comment (go-bears) [reply src/graphql/query/flssLoansQuery.graphql:9]: Whoop! thank YOU! 🎉

**Pull request merged**